live-examples/js-examples/array-splice.html

```diff
@@ -0,0 +1,16 @@
+<pre>
+<code id="static-js">var array1 = ['Jan', 'March', 'April', 'June'];
+
```

Comment:
Could remove this blank line so we fit without a scrollbar. Could also use the variable name  "months".